src/gracetabnote.js

```diff
@@ -0,0 +1,32 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// @author Balazs Forian-Szabo
+//
+// ## Description
+//
+// A basic implementation of grace notes
+// to be rendered on a tab stave.
+//
+// See `tests/gracetabnote_tests.js` for usage examples.
+
+import { TabNote } from './tabnote';
+import { Flow } from './tables';
+
+export class GraceTabNote extends TabNote {
+  static get CATEGORY() { return 'gracetabnotes'; }
+  static get SCALE() { return 0.66; }
```

Comment:
I think this is problematic. Removing the static property means that the scale can't be configured globally. I think this is a more typical use-case than needing to set the scale on a per-instance basis. If you keep the static `GraceTabNote.SCALE` and have `graceTabNote.render_options.scale` default to it, then both global and local styling are available.